lib/ex_unit/test/ex_unit/doc_test_test.exs

```diff
@@ -384,7 +384,7 @@ defmodule ExUnit.DocTestTest do
     output = capture_io(fn -> ExUnit.run end)
 
     # Test order is not guaranteed, we can't match this as a string for each failing doctest
-    assert output =~ ~r/\d+\) test moduledoc at ExUnit\.DocTestTest\.Invalid \(\d+\) \(ExUnit\.DocTestTest\.ActuallyCompiled\)/
+    assert output =~ ~R/\d+\) test moduledoc at ExUnit\.DocTestTest\.Invalid \(\d+\) \(ExUnit\.DocTestTest\.ActuallyCompiled\)/
```

Comment:
Agreed, let's go with `~r` for less changes and because I believe `~R` raises more questions (why is this uppercase?) than what we get out of it (basically nothing). :)